test/UnitTestFramework/UnitTestFramework.js

```diff
@@ -12,12 +12,14 @@
 //     }
 //   How to define and run tests:
 //     var tests = [ { name: "test name 1", body: function () {} }, ...];
-//     testRunner.run(tests);
+//     testRunner.runTests(tests);       // regular tests
+//     testRunner.runAsyncTests(tests);  // tests loaded with WScript.LoadModule

```

Comment:
>testRunner.runAsyncTests(tests);  // tests loaded with WScript.LoadModule [](start = 7, length = 73)

Can this in theory be used with async patterns besides module loading? If it's module-only, should this be named differently?